src/js/components/homepage/visualizations/geo/MapVisualization.jsx

```diff
@@ -126,20 +126,8 @@ export default class MapVisualization extends React.Component {
                         active={this.props.showPerCapita}
                         pressedToggle={this.props.togglePerCapita} />
                 </div>
-
                 <div className="homepage-map-citation">
-                    <p>
-                        The map reflects where obligated funding was used. This is called &quot;place of performance.&quot;
-                    </p>
-                    <p>
-                        Sources: U.S. Census Bureau, Population Division
-                    </p>
-                    <p>
-                        The World Factbook 2013-14. Washington, DC: Central Intelligence Agency, 2013
-                        <span className="info-icon">
-                            <InfoCircle alt="Citation" />
-                        </span>
-                    </p>
+                    <HomepageMapCitation />
```

Comment:
Let's move `className="homepage-map-citation"` into the top-level `div` in `HomepageMapCitation`. Then we can just include `<HomepageMapCitation />` without nesting it in this `<div>`.